src/app/Main/Body/QueryPage/Drawer/Drawer-Layout.component.tsx

```diff
@@ -50,23 +25,40 @@ export function DrawerLayoutComponent(props: {
     drawerTitle: string;
 }): JSX.Element {
     const drawerWidthSting = (props.drawerWidth as unknown) as string;
-    const classes = useStyles(drawerWidthSting);
+
 
     return (
         <Drawer
-            className={classes.drawer}
-            variant="persistent"
-            anchor="left"
-            open={props.isOpen}
-            classes={{
-                paper: classes.drawerPaper,
-            }}
+          sx={{
```

Comment:
Why is the style now defined in the component and no longer with the help of "makeStyles"? In other components makeStyles is still included (inconsistent?).

Reply:
Yeah - I had issues with CSS Specificity & not the right styles taking affect.  I've pushed the version with MakeStyles - we could commit that but the styles need to be adjusted again.  What do you think?

Reply:
Difficult for me to say because I don't see the frontend. But yes we can commit the update and in an extra ticket adjust the style where it is incorrect.